Terrace-Svelte/src/components/Board/Board.svelte

```diff
@@ -18,22 +18,22 @@
         // data stream, and update the current nav
         if (e.srcElement.classList[0] == "Hardware"){
             activeNav = "Hardware";
-            currentNav.update(value => activeNav)
+            // currentNav.update(value => activeNav)
             activeHardwareClient.update(value => e.srcElement.innerText)
             wsSend({"event": "HARDWARE-REQUEST", "requested-client": e.srcElement.innerText})
 
         // Otherwise, update the current nav to reflect the target nav view
         } else {
             activeNav = e.srcElement.innerText;
-            currentNav.update(value => activeNav)
+            // currentNav.update(value => activeNav)
         }
 	}
 </script>
 
     <div id="board">
         <NavBar {navItems} on:click={setView}/>
         {#if (activeNav == "Notes")}
-            <NotesHome />
+            <!-- <NotesHome /> -->
```

Comment:
does not exist as of most recent commit, needs to be pushed.

Reply:
Yeah sorry, I had notes/ in my gitignore from when I was still working with vanilla JS, and didnt realize it wasnt pushing the Notes component, ill get right on that